src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPoolWorkQueue.cs

```diff
@@ -1042,7 +1042,7 @@ public ThreadPoolWorkQueueThreadLocals(ThreadPoolWorkQueue tpq)
             workStealingQueue = new ThreadPoolWorkQueue.WorkStealingQueue();
             ThreadPoolWorkQueue.WorkStealingQueueList.Add(workStealingQueue);
             currentThread = Thread.CurrentThread;
-            threadLocalCompletionCountObject = ThreadPool.GetOrCreateThreadLocalCompletionCountObject();
+            threadLocalCompletionCountObject = ThreadPool.GetOrCreateThreadLocalCompletionCountObject()!;
```

Comment:
The returned type should already be non-nullable
```suggestion
            threadLocalCompletionCountObject = ThreadPool.GetOrCreateThreadLocalCompletionCountObject();
```